There's some problem with temp file, don't know why...

Error During Test
  Test threw an exception of type SystemError
  Expression: readstring(tmp) == "123\n"
  SystemError: opening file /tmp/julia_GolmjG: No such file or directory
  Stacktrace:
   [1] #systemerror#44 at ./error.jl:64 [inlined]
   [2] systemerror at ./error.jl:64
   [3] open at ./iostream.jl:104
   [4] open at ./iostream.jl:150
   [5] macro expansion at test/repl.jl:139 [inlined]
   [6] anonymous at ./<missing>:?
   [7] include_from_node1 at ./loading.jl:569
   [8] include at ./sysimg.jl:14
   [9] macro expansion at test/testdefs.jl:18 [inlined]
   [10] macro expansion at ./test.jl:860 [inlined]
   [11] macro expansion at ./util.jl:378 [inlined]
   [12] macro expansion at test/testdefs.jl:17 [inlined]
   [13] anonymous at ./<missing>:?

--- a/test/repl.jl
+++ b/test/repl.jl
@@ -126,7 +126,7 @@
 
     # issues #22176 & #20482
     # TODO: figure out how to test this on Windows
-    is_windows() || let tmp = tempname()
+    true || let tmp = tempname()
         try
             write(stdin_write, ";")
             readuntil(stdout_read, "shell> ")
